src/UserAccountDataChecker.php

```diff
@@ -63,26 +63,26 @@ public function checkSorbs() {
 		$ip = $wgRequest->getIP();
 		if ( $wgEnableSorbs && !in_array( $ip, $wgProxyWhitelist ) &&
 		  $wgUser->inSorbsBlacklist( $ip ) )
-		 	$this->error( wfMsg( 'sorbs_create_account_reason' ) );
+		 	$this->error( wfMessage( 'sorbs_create_account_reason' )->text() );
 	}
 
 	public function checkUserExists() {
 		if ( $this->mUser->idForName() )
-			$this->error( wfMsg( 'ses-userexists' ) );
+			$this->error( wfMessage( 'ses-userexists' )->text() );
 	}
 
 	public function checkPasswordLength() {
 		if ( !$this->mUser->isValidPassword( $this->mPassword ) )
 		{
 			global $wgMinimalPasswordLength;
-			$this->error( wfMsgExt( 'passwordtooshort', array( 'parsemag' ), $wgMinimalPasswordLength ) );
+			$this->error( wfMessage( 'passwordtooshort' )->numParams( $wgMinimalPasswordLength )->text() );
```

Comment:
Nice. You fixed an i18n bug in the process. Now that you're using numParams(), numeral conversion is also applied automagically.